src/components/index.d.ts

```diff
@@ -74,14 +74,29 @@ export declare const Logo: {
   Inline: typeof LogoInline
 }
 
+interface downstreamProperties {
+  backArrowIcon?: boolean,
+  arrowIcon?: boolean,
+  type: string,
+  isSelected: boolean,
+  fullWidth: boolean,
+  disabled: boolean,
+  name: string,
+  onClick: any,
+  'data-tid': string,
```

Comment:
nit: any reason to make this as string?
(Not blocking, just a curiosity question)

Reply:
Ethos codes has always had these data tids as data attributes with string values used for making tests against so it's of type `:string`